packages/cadl-ranch-specs/http/type/model/empty/main.tsp

```diff
@@ -0,0 +1,24 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates usage of empty model used in operation's parameters and responses.")
+@scenarioService("/type/model/empty")
+namespace Type.Model.Empty;
+
+@doc("Empty model used in operation parameters")
+model EmptyInput {}
+
+@doc("Empty model used in operation return type")
+model EmptyOutput {}
+
+@scenario
+@scenarioDoc("Send a POST request with the following body {}")
+@route("/input")
+op input(@body input: EmptyInput): void;
+
+@scenario
+@scenarioDoc("Send a GET request which return the following body {}")
+@route("/output")
+op output(): EmptyOutput;
```

Comment:
I assume both can be on same endpoint, one as `@get`, anothor `@put`? No need for input/output in endpoint.

e.g. https://github.com/Azure/cadl-ranch/blob/58fdd380c43881abfbd92e9c52adc0d19bd1cef3/packages/cadl-ranch-specs/http/type/property/optional/main.tsp#L20-L69

Reply:
Sure I'll update to the same endpoint.

Reply:
can we add a roundtrip empty model which will be used as both input and output (may or may not in the same operation)

Reply:
@chunyu3 Do you propose to add another op, or just convert the current `@put` to roundtrip (so that it returns 200 + `{}`)?

Reply:
@weidongxu-microsoft  Both are fine.